python/ribasim/ribasim/db_utils.py

```diff
@@ -0,0 +1,66 @@
+from contextlib import closing
+from pathlib import Path
+from sqlite3 import Connection, connect
+
+
+def esc_id(identifier: str) -> str:
+    """Escape SQLite identifiers."""
+    return '"' + identifier.replace('"', '""') + '"'
```

Comment:
Would something like this be more readable?

```suggestion
    identifier = identifier.replace('"', '""')
    return f'"{identifier}"'
```